sim/validator.py

```diff
@@ -169,13 +186,15 @@ def main():
         description="Performance test btb-seq code")
     parser.add_argument("btb_seq", help="path to btb-seq code")
     parser.add_argument("results", help="path to performance test results")
-    parser.add_argument("--ref", "-r", help="path to reference fasta", default=DEFAULT_REFERENCE_PATH)
-    parser.add_argument("--branch", help="path to reference fasta", default=None)
+    parser.add_argument("--branch", help="name of btb-seq branch to use", default=None)
+    parser.add_argument("--predef_snps", help="optional path to VCF file with predefined SNPs", default=None)
+    parser.add_argument("--num_snps", help="number of simulated snps, overridden by predef_snps", default=16000)
```

Comment:
I don't think we need command line arguments for this. I'm about to add functionality to run multiple samples.

Reply:
Oki doki. How do you think we should define whether to use random or predefined SNPs. And which snippy VCF to use?

Reply:
Leave that for the next PR I think, adding support for multiple samples today, will need to integrate with that. If we have functions for doing both, should  be easy to sort that out.

Reply:
gotcha